examples/ckks/bootstrapping/main.go

```diff
@@ -24,20 +24,23 @@ func main() {
 	var plaintext *ckks.Plaintext
 
 	// Bootstrapping parameters
-	// Four sets of parameters (index 0 to 3) ensuring 128 bit of security
-	// are available in github.com/tuneinsight/lattigo/v3/ckks/bootstrap_params
+	// 2xFour sets of parameters (index 0 to 3) ensuring 128 bit of security
+	// are available in github.com/ldsec/lattigo/v2/ckks/bootstrapping/default_params.go
```

Comment:
`github.com/tuneinsight/lattigo/v3/ckks/bootstrapping/default_params.go`

Reply:
Or simply remove the `github.com/ldsec/lattigo/v2/` prefix to avoid having to update it at each major version.